kibana/README.md

```diff
@@ -996,3 +996,64 @@ readonlyrest:
 ```
 
 Now try to login as user1, and ReadonlyREST Enterprise should initialise the index ".kibana\_user1" with all the index patterns and dashboards contained in the template tenancy.
+
+## Impersonation
```

Comment:
shouldn't we have a separate page for the Impersonation feature? Obviously, at this level, we could add info that there is sth like Impersonation, what is this and what scenarios can it be useful for 

(we can extend the first paragraph from https://github.com/beshu-tech/readonlyrest-docs/blob/de65586040c8c20c2f77fa725bdf567d60d6f969/elasticsearch-details/impersonation.md)

Reply:
If I understand correctly, you suggest:

1. Add some basic info about the impersonation feature into the  ReadonlyREST docs kibana section
2. Move the rest into the markdown file on the GitHub 

I understand the motivation, the downside of this approach is the reduction of the ability to search for a specific phrase related to impersonation, like (auth mock service, test settings, free type impersonation, etc.), since the "advance" description will be in the separate GitHub file. Maybe it's a wrong assumption but, it will be hard  for the user to discover, that there is another file with a feature description

I'm not sure if you are talking about the same when you mention "separate page" (probably yes, but better to ask :P ) https://github.com/beshu-tech/readonlyrest-docs/pull/114#issuecomment-1129096503

Reply:
Yes, a small paragraph in main page and a whole page with the details. The search function is global in gitbook anyway.

Reply:
we should take into consideration the fact that the user will read [this](https://docs.readonlyrest.com/), not the GitHub repo markdowns directly

Reply:
Ok, so this [Github markdown page](https://github.com/beshu-tech/readonlyrest-docs/blob/de65586040c8c20c2f77fa725bdf567d60d6f969/elasticsearch-details/impersonation.md) will be the part of the [gitbook](https://docs.readonlyrest.com/).

 Because if I understand correctly, now:
1. Impersonation section is not a part of [gitbook](https://docs.readonlyrest.com/)
2. The only reference to this impersonation description  file is a hyperlink, for example here https://docs.readonlyrest.com/elasticsearch#jwt_auth

Reply:
Correct, let's change that so we have 1/2sentences about impersonation (enough so people ctrl f), and then a full blown page a out it separately

Reply:
Moved all content into impersonation.md file and left a few sentences and links in a gitbook